todo_app/app.py

```diff
@@ -8,8 +9,26 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    todo_items = session_items.get_items()
+    print(todo_items) # adding this line to prove data being pulled from session items 
+    return render_template('index.html', items = todo_items)
 
+@app.route('/add', methods=['GET', 'POST'])
```

Comment:
only needs to support POST
```suggestion
@app.route('/add', methods=['POST'])
```